serveradmin/serverdb/querybuilder.py

```diff
@@ -114,22 +114,16 @@ def get_sql_condition(self, attribute, filt):  # NOQA C901
             else:
                 operator = '&&'
 
-            network_sql_array = "'{{{{{}}}}}'".format(
-                ','.join(str(n) for n in filt.networks)
-            )
-            template = "{{}} {} ANY({})".format(
-                operator, network_sql_array
-            )
-
+            template = "{{}} {} '{}'".format(operator, filt.value)
             if isinstance(filt, InsideOnlyNetwork):
                 template += (
                     ' AND NOT EXISTS ('
                     '   SELECT 1 '
                     '   FROM server AS supernet '
                     '   WHERE {{}} << supernet.intern_ip AND '
-                    '       supernet.intern_ip << ANY({})'
+                    '       supernet.intern_ip << {}'
```

Comment:
Why are we using string concatenation for SQL queries in first place? Bet there is some better solution for that, right? Should have in mind for later to replace those concatenations with some query builder to reduce possibility of SQL injections.

Reply:
Certainly.  We need to change this to at least use parameterized queries.  Good thing is that after this merge request custom SQL generation is restricted to a single Python file only, admittedly not a simple one.